Description: Use correct port number for memcached during tests
 The test attempts to start gearmand and memcached on the same port.  Adjust
 memcached port to match gearmand arguments.
Author: Stig Sandbeck Mathisen <ssm@debian.org>
Last-Update: 2012-03-16
Forwarded: yes
--- a/tests/memcached_test.cc
+++ b/tests/memcached_test.cc
@@ -70,7 +70,7 @@
     return NULL;
   }
 
-  if (server_startup(servers, "memcached", default_port(), 0, NULL) == false)
+  if (server_startup(servers, "memcached", 12555, 0, NULL) == false)
   {
     error= TEST_FAILURE;
     return NULL;
